### DISCUSSION TITLE
How to include PET image information in segmentation?

### DISCUSSION THREAD

**Discussion by suya-li** [ENHANCE-PET/MOOSE#75] [Q&A] [answered]
I tried command: moosez -d <path to dir>  -m clin_ct_organs and want to get organ segmentation results based on FDG-PET/CT image pairs. However, it seems that the segmentation results only depend on CT image, not PET. I tried to put different PET images while keep CT image the same, it gives me the same segmentation every time... 
BTW, the code can recognize PET images in the folder since it is able to output the PET statistics... It simply does not include PET image information to do segmentation.
Does anyone know how to solve this? It will be greatly appreciated...

**Comment by LalithShiyam** [comment]
Hi @suya-li, 
Many thanks for the message.`moosev2` completely ignores PET information when it comes to segmentation. And this was intentional. We used PET for the brain subregion segmentation and we didn't feel it was optimal, so we removed it altogether. 

However `moosev2` recognizes the PET images and extracts the statistics purely for convenience reasons. And I can confirm that we will not include PET information for organ segmentation. Sorry to disappoint you. BTW why do you want the PET information to be considered in the segmentation process, any particular reason?
Cheers,
Lalith

**Reply by suya-li** [reply]
Hi Lalith, thank you for your response. By including PET information in segmentation, we are able to test how PET images from different recon algorithms affect segmentation performance...
Is there anyway I can use moosev1? I remember the PET information is included there. Brain is out of the view in my problem, so I guess maybe moosev1 will work for me.
Also, is there any way to access your registration result? So the mask can be applied to PET images. Since moose is able to output PET statistics, I assume there is a registration step... 
Thank you very much for your help!

Sincerely,
Suya

**Reply by LalithShiyam** [reply, marked as answer]
Hi @suya-li,
I understand your rational better now. But I think you have misunderstood MOOSEV1 and V2. MOOSE V1, doesn't use PET information for organ segmentation. It independently used the PET information without CT only for the brain segmentation part. Organ segmentation was **always** done with CT only. The behavior is **consistent** with V1 and V2. And we will not use PET information in the next versions as well as there are multiple tracers one can use and using only CT is the safe bet. 

Regarding your query on registration, we used a routine reslice operation in V2, which basically resamples CT image to the PT image space. It's not a real image registration operation. Since it's usually from a PET/CT, I think it's a reasonably fair operation to do. 

Aligning PET/CT is not trivial and currently we are working on a different tool to solve this problem. 

Hope I clarified your concerns.

Cheers,
Lalith

**Reply by suya-li** [reply]
Got it! Thank you very much!
